README.md

```diff
@@ -177,6 +177,10 @@ npm run generate
     - Check to see that "Less secure app access" is turned ON. Under "Manage your Google Account" > Go to the security tab. By default, the setting is off for security reasons.
     - If using 2FA make sure to set an [app password](https://support.google.com/accounts/answer/185833?p=InvalidSecondFactor&visit_id=637355441414497566-1310044707&rd=1) and use that in place of your normal login password.
 
+### Multichannel
+
+- **Set [DEFAULT_CHANNEL_SLUG] environment variable.**
```

Comment:
Why do we call it the "default channel slug"? Are there non-default slugs used by the Storefront?

Reply:
@patrys No, currently there is only one.

Reply:
@patrys changed